### PR TITLE
Updating changelog with the required format and old information

### PR DESCRIPTION
**Pull request by talshani**
Need to update the changlog with more information

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,28 +5,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+* Directed Graph for experiment authoring - This is the hallmark feature of Entropy which enables breaking 
+down an experiment into a graph of nodes which can run arbitrary code and pass data from node to node.
+* Resource management - nodes in a graph share resourced which can be, but are not limited to, lab instruments.
+This allows easy usage of resources and sharing of information as well as persistence of state and automated saving
+of metadata representing the state of the entire lab.
+* Data saving backend - Node data, and resource state data, is saved to a DB backend. 
+This release support saving to a SQLite DB. The backend is extendable and additional persistence targets can be added.
+* QCodes driver library adapter - an adapter for the drivers provided by QCodes to be easily used with the
+Entropy persistence backend.
+* QPU-DB - This is an extension to entropy which is built to save and manage the pieced of information describing
```
Comment (galwiner) [inline CHANGELOG.md:34]: We should warn here that we intend to change QPU-DB in the near future

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,28 +5,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+* New HDF5 support
 
 ## [0.1.3]
 ### Added
 * Pass save results as parameter for node
```
Comment (galwiner) [inline CHANGELOG.md:13]: maybe : pass "save results" ... 
to make it more clear that this is a flag for saving, or not saving

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,28 +5,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 
 ## [0.1.2] - 2021-06-08
-
 ### Added
 * Pass save results as parameter for node
```
Comment (galwiner) [inline CHANGELOG.md:17]: duplicate for 0.1.3
was it added there or here?

Comment (talshani) [reply CHANGELOG.md:17]: there is no 0.1.3 :|

**Review by galwiner — APPROVED** [review]

**Pull request merged**